app/javascript/components/nav/components/menu-entry/component.jsx

```diff
@@ -52,7 +52,7 @@ const Component = ({
   const navItemName = name === "username" ? username : i18n.t(name);
 
   const renderNavAction = (
-    <div>
+    <li>
```

Comment:
Does this change how the menu looks? Same for the file above.

Reply:
I need to verify it again. I will do that

Reply:
I checked it and I couldn't see any change in the menu look